src/plugins/base.py

```diff
@@ -19,7 +19,7 @@ def __init__(self, bot):
         """
         self.bot = bot
 
-    def handle_tokens(self, channel, msg, keywords, callback):
```

Comment:
This is so that you can add more arguments like nick, and anything else you'd want to pass to the callback.